test/core/Gatekeeping.t.sol

```diff
@@ -442,18 +415,18 @@ contract GatekeepingTest is IMaker, MangroveTest {
     uint other_ofr = mgv.newOfferByVolume($(quote), $(base), 1 ether, 1 ether, 100_000, 0);
 
     trade_cb = abi.encodeCall(this.updateOfferOK, ($(quote), $(base), other_ofr));
-    uint ofr = mgv.newOfferByVolume($(base), $(quote), 1 ether, 1 ether, 400_000, 0);
```

Comment:
The assertion two lines down (L448) is checking that the gasreq of the offer posted in line 442 has been changed as defined in the posthook. Both numbers are defined in the test, are unaffected by `setUp`, and (I believe) can only be changed by the posthook (the offer id is passed from the test). So it should be ok?

Reply:
Thinking some more on this: I think was we need is an assertion that the expected offer was executed since it's not guaranteed by `marketOrder` whereas it was with `snipe`. The tests in `Posthook.t.sol` should allow us to assume that the posthook is called if the offer is executed.

Reply:
I'm implementing this and it's cleaner + makes things easier to read

Reply:
Addressed in [6d1388d](https://github.com/mangrovedao/mangrove-core/pull/454/commits/6d1388d7700b735d4d362afbe231ab09109b62bd)